### PR TITLE
release review period link

### PR DESCRIPTION
**Pull request by klumb**
Signed-off-by: Kevin Lumbard <kmlumbard@gmail.com>

This is time sensitive so please review and merge before July 1st

### CODE REVIEW

**Comment by foundjem** [comment]
LGTM!

Signed-Of-By: Armstrong <foundjem@ieee.org>

File: metrics/Reviews.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Reviews
 
-Question: What new review requests for changes to the source code occurred during a certain period? 
+Question: What new review requests for changes to the source code occurred during a certain period?
+
+_This metric is a release candidate The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/031/2020. To comment on this metric please see [Issue #337](https://github.com/chaoss/wg-evolution/issues/337). Following the comment period this metric will be included in the next regular release._
```
Comment (GeorgLink) [inline metrics/Reviews.md:5]:
```suggestion
_This metric is a release candidate. The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/31/2020. To comment on this metric please see [Issue #337](https://github.com/chaoss/wg-evolution/issues/337). Following the comment period, this metric will be included in the release._
```

File: metrics/New_Contributors.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Question:  How many contributors are making their first contribution to a given project and who are they?
 
-_This metric is part of the continuous release process and has been through the 30-day review period. However, the information displayed on this page may still be edited prior to the regular release._
+_This metric is a release candidate The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/031/2020. To comment on this metric please see [Issue #329](https://github.com/chaoss/wg-evolution/issues/329). Following the comment period this metric will be included in the next regular release._
```
Comment (GeorgLink) [inline metrics/New_Contributors.md:5]:
```suggestion
_This metric is a release candidate. The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/31/2020. To comment on this metric please see [Issue #329](https://github.com/chaoss/wg-evolution/issues/329). Following the comment period, this metric will be included in the release._

```

File: metrics/Inactive_Contributers.md [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 Question: How many Contributors have gone inactive over a specific period of time?
 
-_This metric is a release candidate The 30 day comment period for this metric begins on 05/06/2020 and ends on 06/06/2020. To comment on this metric please see [Issue #357](https://github.com/chaoss/wg-evolution/issues/357). Following the comment period this metric will be included in the next regular release._
+_This metric is a release candidate The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/031/2020. To comment on this metric please see [Issue #357](https://github.com/chaoss/wg-evolution/issues/357). Following the comment period this metric will be included in the next regular release._
```
Comment (GeorgLink) [inline metrics/Inactive_Contributers.md:5]:
```suggestion
_This metric is a release candidate. The 30 day comment period for this metric begins on 07/01/2020 and ends on 07/31/2020. To comment on this metric please see [Issue #357](https://github.com/chaoss/wg-evolution/issues/357). Following the comment period, this metric will be included in the release._
```

**Pull request merged**